kura/distrib/pom.xml

```diff
@@ -1364,6 +1457,99 @@
             </build>
         </profile>
 
+		<profile>
+            <id>generic-x86_64-nn</id>
+            <activation>
+                <property>
+                    <name>buildAll</name>
+                </property>
+            </activation>
+            <build>
+                <resources>
+                    <resource>
+                        <directory>src/main/resources</directory>
+                        <filtering>true</filtering>
+                    </resource>
+                </resources>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>buildnumber-maven-plugin</artifactId>
+                        <version>1.0</version>
+                        <executions>
+                            <execution>
+                                <phase>validate</phase>
+                                <goals>
+                                    <goal>create-timestamp</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <version>1.7</version>
+                        <executions>
+                            <execution>
+                                <id>generic-x86_64-nn-jars</id>
+                                <phase>prepare-package</phase>
+                                <goals>
+                                    <goal>run</goal>
+                                </goals>
+                                <configuration>
+                                    <target>
+                                        <property name="buildNumber" value="buildNumber" />
+                                        <property name="project.version" value="${project.version}" />
+                                        <property name="project.build.profile" value="${project.build.profile}" />
+                                        <property name="project.build.directory" value="${project.build.directory}" />
+                                        <property name="build.name" value="generic-x86_64-nn" />
+                                        <property name="target.device" value="generic-x86_64-nn" />
+                                        <property name="native.tag" value="x86_64" />
+                                        <property name="kura.os.version" value="raspbian" />
```

Comment:
I think we have the wrong `kura.os.version` on all of the generic profiles, even networking ones...

Reply:
We should check where that property is used and decide what to set in case of generic...